src/ray/common/task/scheduling_resources.h

```diff
@@ -127,6 +127,12 @@ class ResourceSet {
   /// does not exist.
   FixedPoint GetResource(const std::string &resource_name) const;
 
+  /// Return ture if resource_capacity_ constains the specified resource label.
```

Comment:
`Return ture if this resource set constains the specified resource label.`
Do not expose the implementation details in the documentation.